src/runtime_src/core/pcie/windows/alveo/device_windows.cpp

```diff
@@ -1054,8 +1055,19 @@ struct kds_cu_info
 
     auto& stats = (*it).second;
     result_type cuStats;
-    for (unsigned int i = 0; i < stats->CuCount; i++)
-      cuStats.push_back(std::make_tuple(stats->CuInfo[i].BaseAddress, stats->CuInfo[i].Usage, 0));
+    for (unsigned int i = 0; i < stats->CuCount; i++) {
+      data_type data;
```

Comment:
Initialize the structure to a zeros.
```suggestion
      data_type data = {};
```

Reply:
Thanks Stephen for the comment. I will leave this change to Windows guys. They will need to update the CU name line.

Reply:
@rozumx Soren also help test some cases on his side. It looks good.
I have talk to the Windows guys. They said we can go ahead and merge this change.

I will let Max know and merge the change.